docs/use-cases/one-to-many.md

```diff
@@ -0,0 +1,138 @@
+
+- **Public:** Any user inside or outside of your organization will be able to view and use this fragment.
+- **Private:** No user outside of your organization will be able to view or use this fragment.
+- **Unlisted:** Any user inside or outside of your organization, with a direct link, will be able to view and use this fragment.
+
+Click **Save**.
+
+If you want to edit the fragment later, do it from this screen.
+
+{{<imgproc src="/use-cases/one-to-many/new-fragment.png" resize="700x" class="fill aligncenter" style="max-width: 350px" declaredimensions=true alt="app.viam.com/fragment interface">}}
+
+{{% /tablestep %}}
+{{% tablestep %}}
+{{<imgproc src="/use-cases/one-to-many/trash-can.svg" class="fill alignleft" style="max-width: 50px" declaredimensions=true alt="Delete">}}
+**3. Delete the original configuration (optional)**
+
+Now that the configuration is saved as a fragment, you can delete the original config from your machine and _replace it with the fragment_ in the next step.
+In this way, you can keep all your machines up to date whenever you change the fragment.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Add a fragment to multiple machines
+
+With your fragment created, you can add it to as many machines as you'd like:
+
+{{< table >}}
+{{% tablestep %}}
+{{<imgproc src="appendix/try-viam/rover-resources/fragments/fragments_list.png" resize="800x" class="fill alignleft" style="max-width: 300px" declaredimensions=true alt="Add fragment">}}
+**1. Add the fragment to one machine**
+
+On your machine's **CONFIGURE** tab, click the **+** button and select **Insert fragment**.
+Search for your fragment and add it.
+
+Click **Save** in the upper right corner of the screen.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+{{<imgproc src="/use-cases/one-to-many/repeat.svg" class="fill alignleft" style="max-width: 120px"  declaredimensions=true alt="Repeat">}}
+**2. Repeat for each machine**
+
+Repeat step 1 for each of the machines that you want to configure in the same way.
+
+If some of your machines have slight differences, you can still add the fragment and then add fragment mods in the next section.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Modify a fragment
+
+If your machines are similar but not identical, you can use a fragment with all of them, and then [overwrite parts of it](/fleet/fragments/#modify-the-config-of-a-machine-that-uses-a-fragment) to customize select fields of the configuration without modifying the upstream fragment.
+
+{{% alert title="Note" color="note" %}}
+If you modify fields within a fragment, your modifications will act as overwrites.
+If you later update the upstream fragment, the updates to the overwritten fields will effectively not be updated.
+{{% /alert %}}
+
+{{< table >}}
+{{% tablestep link="/fleet/fragments/#modify-the-config-of-a-machine-that-uses-a-fragment" %}}
+{{<imgproc src="/use-cases/one-to-many/edit.svg" class="fill alignleft" style="max-width: 50px"  declaredimensions=true alt="Edit">}}
```

Comment:
And then here maybe show a picture of the edits?

![Screenshot 2024-08-08 at 18 28 21](https://github.com/user-attachments/assets/3da69a30-8544-4965-a14e-4e71811466be)

Reply:
Mm but they won't see this unless they click advanced... I'll add mention of that